src/languages/en.ts

```diff
@@ -1217,8 +1217,10 @@ export default {
         tryAgain: 'Try again',
         verifyIdentity: 'Verify identity',
         genericError: 'There was an error while processing this step. Please try again.',
-        cameraPermissionsNotGranted: 'Camera permissions not granted',
-        cameraRequestMessage: 'You have not granted us camera access. We need access to complete verification.',
+        cameraPermissionsNotGranted: 'Enable Camera Access',
+        cameraRequestMessage: 'We need access to your camera to complete bank account verification. Please enable via Settings > New Expensify',
+        microphonePermissionsNotGranted: 'Enable Microphone Access',
```

Comment:
> - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.

This is already approved but just to double confirm that this is not treated as sentence and capitalization of each word is fine.
cc: @mallenexpensify